src/site/content/en/blog/aem-with-workbox/index.md

```diff
@@ -0,0 +1,138 @@
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/mhq0Kyzn88zfryTFpQgU.png", alt="Configuring the installable experience.", width="800", height="634" %}
+
+
+The start URL is the entry point of your PWA. When a user taps on the the PWA icon on their phone, they will be accessing the startup URL. The display mode configures whether the app is a windowed or fullscreen experience. You can also specify the screen orientation of the application. The theme color is the color of the window and toolbar while the background color is the color of the splash screen when the application is launched. The icon is the image that is shown on the devices home screen or application drawer when the application is installed on the device. The configuration shown in the image generates the manifest JSON shown below.
+
+```json
+{
+  "name": "WKND Adventures and Travel",
+  "short_name": "WKND Adventures and Travel",
+  "start_url": "/content/wknd/us/en.html",
+  "display": "standalone",
+  "theme_color": "#FFDC00",
+  "background_color": "#FF851B",
+  "orientation": "any",
+  "icons": [
+    {
+      "src": "/content/dam/wknd/pwa-logo.png",
+      "type": "image/png",
+      "sizes": "512x512",
+      "purpose": "any maskable"
+    }
+  ]
+}
+```
+
+The start URL can be different from the default landing page for your domain. By changing the `start_url` parameter, you can send your users directly to the application experience rather than the default page a non-authenticated or new user would be presented. This provides PWA users with a smoother, more app-like experience.
+
+AEM understands that different locales can have a different look and feel. You can configure different properties, colors, and icons for different locales or languages and then synchronize the configuration with the linked pages.
+
+Once the PWA is accessed on the browser, you can right click and inspect to bring up DevTools and view the manifest under the Applications panel.
+
+{% Img src="image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/xWxP7YILDgejrdtbq0hh.png", alt="A PWA in the DevTools Application panel.", width="800", height="966" %}
+
+### Configure the service worker
+
+You can configure the content to cache and the caching strategy to use.
+
+If you have been using service workers you may be familiar with [caching strategies](https://developers.google.com/web/tools/workbox/modules/workbox-strategies). Caching strategies specify which resources to cache and whether to look for those resources in the cache first, the network first, or in the cache with a network fallback. You can then choose the resources to pre-cache when the service worker is installed. AEM Application service workers implement a [warm cache strategy](https://developers.google.com/web/tools/workbox/guides/advanced-recipes#warm_the_runtime_cache) which means user's experience will not break even if you specify a missing or broken path.
```

Comment:
```suggestion
If you have been using service workers you may be familiar with [caching strategies](https://developer.chrome.com/docs/workbox/modules/workbox-strategies/). Caching strategies specify which resources to cache and whether to look for those resources in the cache first, the network first, or in the cache with a network fallback. You can then choose the resources to pre-cache when the service worker is installed. AEM Application service workers implement a [warm cache strategy](https://developer.chrome.com/docs/workbox/managing-fallback-responses/#warming-the-runtime-cache) which means user's experience will not break even if you specify a missing or broken path.
```

Reply:
Even though the Workbox docs are only soft launched, we'd like to start driving people over to d.c.c. instead of  WebFundamentals where possible.